web/src/filter.ts

```diff
@@ -1697,6 +1697,10 @@ export class Filter {
         return false;
     }
 
+    is_interleaved_view(): boolean {
+        return !this.can_bucket_by("channel", "topic") && !this.can_bucket_by("dm");
```

Comment:
What's the purpose of this function? I'm not sure it's implementation matches the name -- any view that is a subset of a conversation view is not interleaved, for example.